### PR TITLE
Add test case for read in envtools

### LINKED ISSUE #2615: Implement `envtool shell read` subcommand
### What should be done?

It should read the given file and print its content to stdout.

Then use it instead of `cat` command in the Taskfiles.

### Where?

* https://github.com/FerretDB/FerretDB/blob/main/Taskfile.yml
* https://github.com/FerretDB/FerretDB/tree/main/cmd/envtool

### Definition of Done

* envtool updated;
* Taskfile.yml updated.

### PR DESCRIPTION
**Pull request by wqhhust**
# Description

Closes #2615.

## Readiness checklist

* [x] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [ ] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [ ] I marked all done items in this checklist.

### CODE REVIEW

File: cmd/envtool/envtool_test.go [hunk 1]
```diff
@@ -39,6 +41,17 @@ func TestRmdirAbsentDir(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestRead(t *testing.T) {
+	t.Parallel()
+	s := "test string in a file"
+	f := "file.temp"
+	err := os.WriteFile(f, []byte(s), 0o644)
```
Comment (chilagrow) [inline cmd/envtool/envtool_test.go:48]: Let's use `t.TempDir()` directory. The directory will be removed automatically when the test completes, so the file won't exist after the test. :hug

Also `os.CreateTemp()` is quite useful for creating temporary file, like for testing. It creates a unique filename so the same test can be run concurrently.

**Review by chilagrow — COMMENTED** [review]
Thanks for adding the test! Just have a minor comment

**Comment by chilagrow** [comment]
Also could you select what you've done on **Readiness checklist** in https://github.com/FerretDB/FerretDB/pull/2657#issue-1714727154 please :pray:

**Review by chilagrow — APPROVED** [review]
Thanks for your contribution again!

**Review by noisersup — APPROVED** [review]
LGTM

**Pull request merged**